docs/section22/Rule22-17.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-17  
+
+**Schema Version:** 0.0.12 **Mandatory Rule:** True  
+**Rule ID:** 22-17  
+**Rule Description:** The baseline condenser-water design supply temperature shall be calculated using the cooling tower approach to the 0.4% evaporation design wet-bulb temperature, valid for wet-bulbs from 55°F to 90°F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.11 Heat Rejection (System 7, 8, 11, 12 and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Function Calls:**  
+
+1. get_component_by_id()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each heat rejection device in B_RMR: `for tower_b in B_RMR.RulesetModelInstance.heat_rejections:`
+
+  - Get reported design wet-bulb temperature: `wbt_b = tower_b.design_wetbulb_temperature`
+
+  - Get reported approach temperature: `approach_b = tower_b.approach`
+
+  - Get condenser water loop served by heat rejection device: `cw_loop_id_b = tower_b.loop`
+
+    - Get design supply temperature of condenser loop: `supply_temperature_b = get_component_by_id(cw_loop_id_b).cooling_or_condensing_design_and_control.design_supply_temperature`
+
+    **Rule Assertion:**
+
```

Comment:
question: should we be checking if wet bulb is in between 55 and 95 as an applicability check as well?
i know we discussed this but I forget what the outcome was.

Reply:
I added an applicability check and a note regarding models with multiple heat rejection devices.